app/javascript/groceries/components/StoreHeader.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="pug">
-h1.h2.store-name.my-4
+h1.h2.my-1
```

Comment:
By changing `my-4` to `my-1` we are able to reclaim a lot of the vertical space otherwise lost by adding the new header.